dev/source/docs/learning-ardupilot-uarts-and-the-console.rst

```diff
@@ -5,35 +5,126 @@ UARTs and the Console
+
++-------------+---------------------------+----------+-------------------------+
+| ParamPrefix | Sim_vehicle Cmd Line      | Def Role | Default Connection      |
++=============+===========================+==========+=========================+
+| \SERIAL0_   | \- -uartA= or \- -serial0 | Console  | tcp:localhost:5760:wait |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL1_   | \- -uartC= or \- -serial1 | MAVLink  | tcp:localhost:5762      |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL2_   | \- -uartD= or \- -serial2 | MAVLink  | tcp:localhost:5763      |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL3_   | \- -uartB= or \- -serial3 | GPS      | Simulated GPS           |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL4_   | \- -uartE= or \- -serial4 | GPS      | Simulated GPS           |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL5_   | \- -uartF= or \- -serial5 |          |                         |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL6_   | \- -uartG= or \- -serial6 |          |                         |
++-------------+---------------------------+----------+-------------------------+
+| \SERIAL7_   | \- -uartH= or \- -serial7 |          |                         |
++-------------+---------------------------+----------+-------------------------+
 
 If you are writing your own sketch using the ArduPilot HAL then you can
 use these UARTs for any purpose you like, but if possible you should try
 to use the above assignments as it will allow you to fit in more easily
 to existing code.
 
-You can change the role of UART by changing its SERIALn_PROTOCOL param. Possible parameter values are 
--1:None, 1:MAVLink1, 2:MAVLink2, 3:Frsky D, 4:Frsky SPort, 5:GPS, 7:Alexmos Gimbal Serial, 8:SToRM32 Gimbal Serial, 9:Rangefinder, 10:FrSky SPort Passthrough (OpenTX), 11:Lidar360, 13:Beacon, 14:Volz servo out, 15:SBus servo out, 16:ESC Telemetry, 17:Devo Telemetry, 18:OpticalFlow, 19:RobotisServo, 20:NMEA Output, 21:WindVane, 22:SLCAN
-Search code for 1_PROTOCOL to get updated list of uart roles.
+You can change the role of UART by changing its SERIALn_PROTOCOL param. Possible parameter values are:
+
+
++-------+----------------------------------+
```

Comment:
This should just be linked to the parameters

Reply:
Done! I wasn't sure that would work. I think it works right now because Periph is in the dev docs. Not sure if that would work without a parameter.rst in the dev wiki.